server/index.js

```diff
@@ -15,6 +16,9 @@ app.prepare().then(() => {
   const server = express()
   server.use(helmet())
   server.use(compression())
+  server.use(cookieParser())
+  server.use(express.urlencoded())
+  server.use(express.json())
```

Comment:
@tranlehaiquan  Can you help explain the purpose of this middleware?

Reply:
On the first time, I thought, it will be easy if we use cookieParser. But maybe another pull request :). Urlendcoded and JSON middleware for endpoints in the future, you may not need it now.

Reply:
The reason you should use cookieParser
![image](https://user-images.githubusercontent.com/17347993/68000454-ca3e0e00-fc92-11e9-91d7-f6e0fa8f592b.png)

Reply:
Nice to know this @tranlehaiquan, but I just not seen the implementation of the cookie-parser.
and how to re-produce the error?

Reply:
Yes, and I remove it middleware. Maybe fix later. @arrlancore